10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
+Below are proposed spreadsheets for use by the 10X techs (not all experiment-related columns are shown).
+
+## FASTQ-Level Spreadsheet
+Laura is working on making a new "FASTQ-oriented" spreadsheet, but that nomenclature is tricky: we're not considering individual run- and read-level FASTQs, but are referring to such a group collectively as a "FASTQ" or "set of FASTQs".
+
+Below is the in-progress outline for this spreadsheet, along with some example configurations:
 
-### FASTQ-Level Spreadsheet
 | FASTQs  | Loading Sample | Hashed Sample | 10x Lane ID | Library Type | BCL Run ID  | Processing Run  |   
 |---|---|---|---|---|---|---|
 | H1_GEX_BCL-1 | H1_GEX  | False  | 1  | GEX  | BCL-1 | CR-1 |
 
-`FASTQs` is the name of the set of FASTQs generated by Cell Ranger Count - this is composed of the loading sample name appended with the `BCL Run ID`. This allows us to handle the common scenario where the same sequencing library (e.g. tube of liquid that can be aliquoted from) is sequenced more than once. `Loading Sample` is the name of the sample that is loaded into a 10x chip lane (e.g. can consist of several samples via hashing). `Hashed Sample` is a True/False value indicating whether hashing has been done (will be redundant with the `Loading Sample` naming convention). `10x Lane ID` will indicate the specific lane number a sample is loaded into - this is necessary for situations where the same sample is loaded into several lanes and in that case will become part of the FASTQs name. `Library Type` indicates what type of library is being prepared (e.g. GEX, ADT; however the chemistry and version may or may not be included ... GEX_5-prime in this column). `BCL Run ID` is the name of the BCL file the FASTQs will be put into. `Processing Run` is the name of the processing run that the data is being organized under - a processing run consists of the jobs necessary to convert BCL inputs into FBM and TCR/VDJ outputs). 
-
-### Sample-Level Spreadsheet
-| Sample name  | Loading Sample |  Sample-Meta-Data ... | HTO | Expected Cell Number | Library Features | 
-|---|---|---|---|---|--|
-| S1_GEX  | H1_GEX | ... | HTO-1  | 5000 | LF-1 |
-
-A CITE-seq run sequencing-library will, in general, contain ADTs and HTOs. The list of features assocaited with this sequencing library are stored in a separate `Library Features Table` (see below and foreign keys under the `Library Features` column). For now, we are omitting the `Sample-Meta-Data` which will include Cell Ranger necessary information like organism, reference transcriptome etc. 
-
-### Library Features Table
-| Library Features| Index |
-|---|---|
-| HTO-1  | HTO-Index-1 |
-| CD8_HIMC-1_Lot-1  | ADT-Index-1 |  
+- `FASTQs`: name of the "set of FASTQs" that will be fed to a single `cellranger count` run.
```

Comment:
> [a] "set of FASTQs" that will be fed to a single `cellranger count` run.

is this an accurate attempt at distilling our "set of `FASTQ`s" concept?

can move it up higher if so, and maybe it will inspire another name we can use.

Reply:
I think we need to improve our nomenclature since there are (at least) two types of "sets of FASTQs". 

The first "set of FASTQs", we discussed earlier is a set of FASTQs de-multiplexed from a BCL that has information on a single sample but is broken into very granular sequencer-lane and read-level FASTQs. The second "set of FASTQs" refers to multiple sets (previous definition) of FASTQs from different sequencing runs on the same sample that need to all be fed into Cell Ranger Count to produce a single FBM (feature barcode matrix). This scenario can arise if a single pooled library was sequenced more than once to obtain more reads or because the first sequencing run failed. 

For clarification, the pooled library is an actual tube of DNA molecules in suspension that have various barcodes/indexes (e.g. cell barcode, UMI barcode, sequencing index) within them and can be repeatedly sampled from (aliquoted) and loaded into to more than one sequencing run. The sequencing run can be thought of as a random sampling of the molecules from a set of cells. 

So - we probably want a new name for the second type.

Reply:
cool, trying to paraphrase:
- each of the two 10x images depicts two of your "first type" of "set of FASTQs"
- the number of "second type" sets corresponds to the number of output FBMs (so 2 and 1 in the two 10x images, resp.)

I thought we were mostly emphasizing the "second type" (and meant to basically say that here on line 30), but I think I understand now that for interfacing w/ sequencing folks, they are most concerned with the "first type".

Does that seem right?

Reply:
Yes to the first bullet and final comment. On the second bullet, the "second type" should not correspond to the number of FBMs but the number of sequencing runs. We should probably refer to the two FASTQ sets as:
1) FASTQ set from a single BCL run (collectively are used to generate a single FBM)
2) FASTQ set from multiple BCL runs (collectively are used to generate a single FBM)